### ISSUE TITLE
[BUG] Division by 0 at startup

### ISSUE THREAD

**Issue by Vincent-HD** [aristocratos/bashtop#49]
**Describe the bug**
When launching bashtop an division by zero occur line 1590

**Info (please complete the following information):**
 - Linux distribution and version : 
Ubuntu Server 18.04 bionic

 - Bash version (version 4.4 or above is required) : 
version 4.4.20(1)-release 

**Additional context**
error.log : 
```
New instance of bashtop version: 0.8.14 Pid: 11032
10:51:33 ERROR: On line 666
/usr/bin/bashtop: line 1590: ( ( 6638-6530 ) * 1000 * 1000 ) / ( cpu[hz]*time_elapsed*cpu[threads] ) : division by 0 (error token is "( cpu[hz]*time_elapsed*cpu[threads] ) ")
```

**Comment by aristocratos** [comment]
Would you mind running `bashtop --debug` and then pasting the resulting "$HOME/.config/bashtop/tracing.log"

**Comment by andresth** [comment]
Hi,
i have the same problem as @Vincent-HD, but my Ubuntu is running in a Parallels container.
It seems that `lscpu` reports 0 CPUs when running in a container.
The `nproc` command returns the correct number of CPUs.

[tracing.zip](https://github.com/aristocratos/bashtop/files/4570180/tracing.zip)

**Comment by aristocratos** [comment]
Sure can add so if cpu output from `lscpu` is 0 then use `nproc`, but also need a secondary way of getting number of threads in that case, since `nproc` only gives number of cores.

**Comment by aristocratos** [comment]
Can you run `cat /sys/devices/system/cpu/present` and post the output?

**Comment by andresth** [comment]
The file `/sys/devices/system/cpu/present` doesn't exist on my server, but there is a file called `/sys/devices/system/cpu/possible` which contains `0-3`

Btw I don't think there is a difference between cores and threads in a container environment.
You could check for the Virtualization type and if it says container use nproc for cores and threads

**Comment by andresth** [comment]
I collected the outputs from other visualizers.

[cpu_possible_docker_rpi.txt](https://github.com/aristocratos/bashtop/files/4570403/cpu_possible_docker_rpi.txt)
[cpu_possible_docker_x64.txt](https://github.com/aristocratos/bashtop/files/4570404/cpu_possible_docker_x64.txt)
[cpu_possible_hyper_v.txt](https://github.com/aristocratos/bashtop/files/4570405/cpu_possible_hyper_v.txt)
[cpu_present_docker_rpi.txt](https://github.com/aristocratos/bashtop/files/4570406/cpu_present_docker_rpi.txt)
[cpu_present_docker_x64.txt](https://github.com/aristocratos/bashtop/files/4570407/cpu_present_docker_x64.txt)
[cpu_present_hyper_v.txt](https://github.com/aristocratos/bashtop/files/4570408/cpu_present_hyper_v.txt)
[lscpu_docker_rpi.txt](https://github.com/aristocratos/bashtop/files/4570409/lscpu_docker_rpi.txt)
[lscpu_docker_x64.txt](https://github.com/aristocratos/bashtop/files/4570410/lscpu_docker_x64.txt)
[lscpu_hyper_v.txt](https://github.com/aristocratos/bashtop/files/4570411/lscpu_hyper_v.txt)

**Comment by aristocratos** [comment]
Fixed in v0.8.23
